solutions/src/test/java/ru/sberbank/school/task02/CalculatorTest.java

```diff
@@ -51,15 +51,15 @@ void convertReturnsIfQuoteVolumeSizeDouble() {
     @Test
     void convertIfProvidedNegativeAmount() {
         FxConversionService calculator = factory.getFxConversionService(quotesService);
-        BigDecimal offer = calculator.convert(ClientOperation.BUY, Symbol.USD_RUB, BigDecimal.valueOf(-1));
-        Assertions.assertNull(offer);
+        Assertions.assertThrows(IllegalArgumentException.class,
+                ()->calculator.convert(ClientOperation.BUY, Symbol.USD_RUB, BigDecimal.valueOf(-1)));
     }
 
     @Test
     void convertIfProvidedZeroAmount() {
         FxConversionService calculator = factory.getFxConversionService(quotesService);
-        BigDecimal offer = calculator.convert(ClientOperation.BUY, Symbol.USD_RUB, BigDecimal.ZERO);
-        Assertions.assertNull(offer);
+        Assertions.assertThrows(IllegalArgumentException.class,
+                ()->calculator.convert(ClientOperation.BUY, Symbol.USD_RUB, BigDecimal.valueOf(-1)));
```

Comment:
WhitespaceAround: &#39;-&gt;&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-483996126>*